src/App.tsx

```diff
@@ -1,24 +1,238 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+/* eslint-disable max-len, jsx-a11y/control-has-associated-label */
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoItem } from './TodoItem';
 
-const USER_ID = 0;
+type Status = 'all' | 'active' | 'completed';
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.